lib/rucio/daemons/hermes/hermes.py

```diff
@@ -517,6 +517,38 @@ def aggregate_to_influx(
     return 204
 
 
+def build_message_dict(
+        bulk: int,
+        worker_number: int,
+        total_workers: int,
+        message_dict: dict,
+        message_ids: list,
+        logger: "LoggerFunction",
+        service: str = ""
+) -> None:
+    start_time = time.time()
+    messages = retrieve_messages(
+        bulk=bulk,
+        old_mode=False,
+        thread=worker_number,
+        total_threads=total_workers,
+        service_filter=service
+    )
+
+    if messages:
+        if service not in message_dict:
+            message_dict[service] = []
+            for message in messages:
+                message_dict[service].append(message)
+                message_ids.append(message["id"])
```

Comment:
I'm not sure what `message_ids` is doing. We append IDs to it but we don't print it out or use it anywhere; this is the case in the current master branch too, it doesn't seem to do anything: https://github.com/rucio/rucio/blob/3d5cbf5d8c0e37f793bbd6f01d1064e44cfe0b4f/lib/rucio/daemons/hermes/hermes.py#L604

should we just remove it?